webapp/components/create_post.jsx

```diff
@@ -698,6 +766,14 @@ export default class CreatePost extends React.Component {
                     show={this.state.showPostDeletedModal}
                     onHide={this.hidePostDeletedModal}
                 />
+                <ConfirmModal
+                    title={notifyAllTitle}
+                    message={notifyAllMessage}
+                    confirmButton={notifyAllConfirm}
+                    show={this.state.showConfirmModal}
+                    onConfirm={this.handleNotifyAllConfirmation}
+                    onCancel={() => this.setState({showConfirmModal: false})}
```

Comment:
Can you make this an actual method of `CreatePost` instead of using an error function? That causes it to make a new function on each call to render